experimental/packages/opentelemetry-sdk-metrics-base/src/view/Aggregation.ts

```diff
@@ -24,19 +28,91 @@ import { InstrumentDescriptor } from '../InstrumentDescriptor';
  * Aggregation provides a set of built-in aggregations via static methods.
  */
 export abstract class Aggregation {
-  // TODO: define the actual aggregator classes
-  abstract createAggregator(instrument: InstrumentDescriptor): unknown;
+  abstract createAggregator(instrument: InstrumentDescriptor): Aggregator<Maybe<Accumulation>>;
 
-  static None(): Aggregation {
-    return NONE_AGGREGATION;
+  static Drop(): Aggregation {
+    return DROP_AGGREGATION;
+  }
+
+  static Sum(): Aggregation {
+    return SUM_AGGREGATION;
+  }
+
+  static LastValue(): Aggregation {
+    return LAST_VALUE_AGGREGATION;
+  }
+
+  static Histogram(): Aggregation {
+    return HISTOGRAM_AGGREGATION;
+  }
+
+  static Default(): Aggregation {
+    return DEFAULT_AGGREGATION;
   }
 }
 
-export class NoneAggregation extends Aggregation {
+export class DropAggregation extends Aggregation {
+  static kDefault = new DropAggregator();
   createAggregator(_instrument: InstrumentDescriptor) {
-    // TODO: define aggregator type
-    return;
+    return DropAggregation.kDefault;
+  }
+}
+
+export class SumAggregation extends Aggregation {
+  static kDefault = new SumAggregator();
+  createAggregator(_instrument: InstrumentDescriptor) {
+    return SumAggregation.kDefault;
+  }
+}
+
+export class LastValueAggregation extends Aggregation {
+  static kDefault = new LastValueAggregator();
+  createAggregator(_instrument: InstrumentDescriptor) {
+    return LastValueAggregation.kDefault;
+  }
+}
+
+export class HistogramAggregation extends Aggregation {
+  static kDefault = new HistogramAggregator([0, 5, 10, 25, 50, 75, 100, 250, 500, 1000]);
```

Comment:
Are these default bucket values specified?

Reply:
Yes, it is specified here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#histogram-aggregation

I'd add a link in the comment for reference.